Update data concept after work shop discussion (#32)
orcestra-campaign/book: orcestra_book/data_concept.md

* The dataset **should** be accessible without credentials.
* The dataset **should** be available publicly as soon as possible (ideally immediately after acquisition).
* The dataset **should** be available publicly not later than one year after the campaign finished. (**TODO:** this point might be better placed in the data policy, we really want something like a "must" here, but in this section this would technically mean that no data can be added after a year, which we also don't want)
The dataset **must** be stored in one the following data formats:
  * NetCDF
  * Zarr
  * CSV
  * GRIB

>   * AMES FFI

Author: My understanding was that these three formats are "fine" as they should be readable in most languages. But I am also very fine with removing them :)

* It **can** be available in different forms in addition
* The list of accepted data formats **can** be extended, if the data format is well standardized and readable by several common programming languages

### 3. datasets are **well-formed** and **analysis-ready**

* You **should** follow standard metadata schemes ([CF-Conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html))
* You **may** designate data processing levels to help users understand the quality of the provided data. If you provide processing levels, the levels **should** follow the [EOSDIS data processing levels](https://www.earthdata.nasa.gov/engage/open-data-services-and-software/data-information-policy/data-levels) scheme.
* You **should** work with your own **published** datasets.
* Datasets **should** be reviewed across teams.

### 4. incremental backups are possible

> * You **must** provide a version number for your dataset

Author: The first would have been my idea. The latter would be fine. How about 
```suggestion
* You **must** provide a version number for your dataset in its catalog metadata
* You **should** provide a version number for your dataset in the dataset attributes
* You **must not** provide any version information in the dataset name
```

* You *should* add a content identifier for your dataset (specify has algorithm?)
* The catalog entry **should** point to the most recent version of the dataset

> * The storage location **may** provide a method to check if something changed ( e.g. HTTP ETag, ...)

Reviewer: ```suggestion
* The storage location **should** provide a method to efficiently check if something changed (e.g. HTTP [ETag](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#etagif-none-match), [If-Modified-Since](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#if-modified-since)...)
```

It's pretty standard practice an not that hard to implement, and it's really really helpful afterwards, so it should at least be a **should**.

### 5. datasets are on a shared, distributed system

Use a distributed storage protocol to make datasets accessible e.g. [IPFS](https://ipfs.tech), [ONEDATA](https://onedata.org)

> :::{info}
> The implementation of this proposal is likely not ready in time.
> However, further investigation and feasibility studies may be carried out during the campaing.
> :::

Reviewer: ```suggestion
```
I wouldn't be so pessimistic...
There's apparently more support coming up at least from DKRZ and possible HALO-DB will also shift towards this direction. Also we are accessing EUREC4A data this way routinely (e.g. in CI).
We however should examine which way to go. IPFS and ONEDATA would be quite different approaches, also ONEDATA seems to be quite a bit more effort to set up.

Author: I am not as optimistic as you, but maybe we shouldn't give up in the concept already :)

A data catalog in our sense is a somewhat formalized way of listing datasets and a method to access those datasets.
A catalog is machine readable and supports the goal of dataset accessibility.

> :::{admonition}

Reviewer: I think this somehow broke the CI...
